### ISSUE TITLE
Merge with upstream?

### ISSUE THREAD

**Issue by htgoebel** [Kozea/ghostscript#1]
Hi, I wonder if you are interested in merging with upsteam? Since I'm no longer used to mercurial, I'd convert the repo to git anyway. What do you think?

**Comment by liZe** [comment]
Sure! There's probably nothing more here now that Python 3 support has been added upstream. As we're using the package in production, the only thing we miss is a 0.5 release to replace [python-3-ghostscript](https://pypi.python.org/pypi/python3-ghostscript/0.5.0).

**Comment by htgoebel** [comment]
> now that Python 3 support has been added upstream

Oh, which upstream we are talking about? I mean <https://bitbucket.org/htgoebel/python-ghostscript>, (maintained (and neglected) by me) and I can't remember I implemented Python 3 support. Which upstream to you mean?

**Comment by liZe** [comment]
> I can't remember I implemented Python 3 support. Which upstream to you mean?

Yes, that's the upstream I was thinking about, and [this commit](https://bitbucket.org/htgoebel/python-ghostscript/commits/334d9b7625866ab92178e31617c095ba4cc58ecc) (and others) lead me to think that Python 3 was now supported. Was this just a change in the `README` and `CHANGES` files?

**Comment by htgoebel** [comment]
To be honest, I can't remember whether I tested it. But at least something like bea55dbdb42fa6449 is missing in my repo. I'll have a look at it the next days and prepare a new release.

I also plan moving to https://gitlab.com/pdftools where I already have some other pdf-related packages. Would you join there a maintainer?

**Comment by liZe** [comment]
> To be honest, I can't remember whether I tested it.

Tests on Linux probably work on Python 3 with imports that you fixed in [`a70e8b3`](https://bitbucket.org/htgoebel/python-ghostscript/commits/8e70eb8e2edc5d9f1229764867a70e8b3e89b4bf).

> But at least something like bea55db is missing in my repo.

You can safely cherry-pick or merge bea55db, that's only minor changes.

> I'll have a look at it the next days and prepare a new release.

Good news!

> I also plan moving to https://gitlab.com/pdftools where I already have some other pdf-related packages. Would you join there a maintainer?

I'm probably not able to add anything interesting more that the 2to3, adding me as a maintainer wouldn't be that useful :wink:.

**Comment by htgoebel** [comment]
Just a quick info: Converting the mercurial repo to git failed (I tried several tools, but still some are left). So the new git repo will take some time.

**Comment by htgoebel** [comment]
After fixing some bugs in reposurgeon (which I use to convert repos to git), I now moved the source-code to gitlab (including CI tests there) and  released a new upstream version.

I suggest you put a pointer to the new repo into the README, and correct the homepage URL.

Thanks for supporting python-ghostscript :-)
